conda_smithy/configure_feedstock.py

```diff
@@ -328,9 +322,7 @@ def _get_used_key_values_by_input_order(
         for key in all_used_vars
         if key in squished_input_variants
     }
-    logger.debug(
-        "initial used_key_values {}".format(pprint.pformat(used_key_values))
-    )
+    logger.debug(f"initial used_key_values {pprint.pformat(used_key_values)}")
```

Comment:
In general, yes. And there is even a ruff rule to check that (which is currently disabled): [G004](https://docs.astral.sh/ruff/rules/logging-f-string/)

I personally prefer f-strings for logging because I think they are easier to read, and usually there should not be much performance to gain because calculating the string representations of the objects should be fast anyway.

I found https://stackoverflow.com/a/54368109, so it seems like I am in a minority, so if you want to, I am totally open to adding the G-rules enforcing logging with deferred positional arguments. Let me know what you think.

Reply:
No strong feelings here. My style is usually deferring as long as possible in logging out of habit so this reads weird, but I'm not going to oppose. Whatever we use, we should be consistent though. So pick one and enforce a rule, it's what I'd say.

Reply:
Alright, let's enforce the G rules then.

Reply:
5dd60c1ccaa8eaa6e0c4a1d24972b50541dff88f